internal/init/prompts.go

```diff
@@ -232,3 +259,22 @@ func appendToSet(set []string, toAppend []string) []string {
 	}
 	return set
 }
+
+// validation fuctions that field validation of type:fuction in zero-module.yaml can select
+// need to implement type:function flow in PromptModuleParams before use
+func validateRootDomain(input string) error {
```

Comment:
Do we still need these now that the modules define their own regexes?

Reply:
right! I kept it as an example ref for when modules can select functions instead of using regexes, it's not necessary. I okay with removing it?

Reply:
Yeah let's remove it.